about.md

```diff
@@ -4,18 +4,53 @@ title: About
 is_nav: true
 permalink: /about/
 ---
+開発、改善、海賊
 
+Astral Dynamics is an Innovation and Rapid Proto-typing agency. We are a multi-
+disciplinary team that brings genuine creativity together with exception 
+technical knowledge and skill. We respond to challenges, design solutions and rapid prototype.
 
-Astral Dynamics is a forward thinking tech startup with a focus on improving and enhancing culture.
+We follow our own Agile methodology and have a unique organisational culture based 
+on the naval metaphore. Every project is a voyage of discovery. We believe very 
+strongly in our freedom and while we occasionally hire out our servives to worthy
+causes, we don't compromise on our ideals or sell what we hold sacred.
 
-We're normally UK based, with our offices in North Wales, but this year we decided to take our company out on the road in Asia --- read more about our adventure [here][1].
+Our home base is in Snowdonia in North West Wales. An [magical][1] place, home 
+to many secrets of the world long forgotten. Here Tolkien came to find his 
+inspiration and here the original language of Britain is still spoken.  On the 
+slopes of Elidir, 8 miles from the [ancient][2] port of [Caernarfon][3], we're 
+building our ship, the Astral, inside 19 Century Welsh [Chapel][4]. Though 
+credit was rarely given, this region has always produces the bravest of
+[sailors][7] and [pirates][8]. Our voyages are on Astral seas of Technology 
+and Innovation. We wield the magic of web; HTML5, React, Angular and NodeJS. 
+Our quests are to enable new information driven cultures and affect behaviour change.
 
-We recently launched our first spin-out company, [Elidir Health][2] who are continuing our relationship with the medical industry. You can find more information on our [projects page][3].
+We're building a global team of high-motivated sailors, ready to 
+tackle grand societal challenges. We firmly believe in the futility of being 
+rich on dying planet and see money only as an enabler as we sail 
+uncompromisingly to create out dreams. All our crew are highly motivated and 
+we have no set working hours, play is both important and productive and 
+enjoying our work is at the core of what we do. When the seas are rough and 
+deadlines tight we will work without sleep and with military precision to 
```

Comment:
military precision sounds exciting and edgy, but it's not accurate. We can't sell ourselves on an expectation that we can't (and don't) live up to.